src/selectors/clusterSelectors.js

```diff
@@ -123,6 +123,9 @@ export const selectAndProduceAZGridTemplateAreas = () =>
 export const selectCanClusterUpgrade = (state, clusterID, targetVersion) => {
   const cluster = state.entities.clusters.items[clusterID];
 
+  if (!cluster) return;
```

Comment:
I believe this should at least return `null` if nothing is found

From ECMAScript2015 spec:

> **4.3.10 undefined value**
> primitive value used when a variable has not been assigned a value
>
> **4.3.12 null value**
> primitive value that represents the intentional absence of any object value

That was my thinking when I introduced that linter rule

Reply:
I am thinking that if we have to return somethiung here, maybe we should return `false` as `canClusterUpgrade` returns a boolean